deploy/runtime/ui/Battery Voltage.txt

```diff
@@ -1361,7 +1756,7 @@ function cell_voltage_vs_dod()
 			count++;
 			t = i*0.1;
 			q0 = Qfull - I*t;
-			V = E0 - R*I - K*(Qfull/(Qfull-I*t)) + A*exp(-B*I*t);
+			V = E0 - R*I - K*(Qfull/(Qfull -I*t)) + A*exp(-B*I*t);
```

Comment:
Should this be changed to batt_Qfull_cutoff? It would be great if we could get the blue line to hit 100% on the chart (assuming that matches the behavior of the battery in SSC.)